### ISSUE TITLE
[Question] Is there a way to use multiple profiles at once?

### ISSUE THREAD

**Issue by Toadfield** [kozec/sc-controller#629]
For example if a game only supports dpad for movement,but I also want to be able to use the analog stick,then can I do multiple profiles,one for dpad and one for analog stick and use both?

**Comment by kozec** [comment]
No, not currently. But you can modify existing profile to have both mappings or have button to switch between profiles on the fly.
